src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -69,15 +65,34 @@ public LogzioSender(String logzioToken, String logzioType, int drainTimeout, int
         queueDirectory = new File(bufferDir);
 
         if (additionalFields != null) {
-            this.additionalFields = Splitter.on(';').omitEmptyStrings().withKeyValueSeparator('=').split(additionalFields);
+
+            JsonObject logMessage = formatMessageAsJson(new Date().getTime(), "Level", "Message", "Logger", "Thread");
```

Comment:
When reading this line you can no clue to why you are creating it in the initialization phase. Maybe naming this var as reservedFieldsTestLogMessage might help? Or maybe exampleMsg or dummyMsg?